contents/blog/the-posthog-array-1-24-0.mdx

```diff
@@ -0,0 +1,350 @@
+- **New:** Advanced dashboard collaboration features
+- **Improvement:** Session recording support for heavy websites
+- **Improvement:** Multiple new configuration options for `posthog-js`
+
+<ArrayCTA />
+
+
+## PostHog 1.24.0 Release Notes
+
+> If you're self-hosting and want to upgrade for a better experience and new features, remember to [update your PostHog instance](/docs/configuring-posthog/upgrading-posthog).
+
+### [GeoIP plugin for all](https://github.com/PostHog/posthog/pull/3894)
+
+![GeoIP](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/geoip.png)
+
+Plugins are now live on PostHog Cloud, and, as a bonus, we have now added out-of-the-box support for the PostHog GeoIP plugin, which adds location properties to your events, such as country and city, as well as a dozen other values!
+
+The plugin works on both cloud and self-hosted installations (`1.24.0` minimum).
+
+### [New 'Cohorts' tab on person pages](https://github.com/PostHog/posthog/pull/3744)
+
+![Cohorts tab](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/person-cohorts.png)
+
+When viewing a person's page in PostHog, you can now toggle between a view of their properties and a view of the cohorts this person is in, giving you a lot more context on the user you're looking at.
+
+### [Toolbar support for custom data attributes](https://github.com/PostHog/posthog/pull/3761)
+
+![Data attr](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/data-attr.png)
+
+To make the experience of using the PostHog toolbar better, we have recommended that you set `data-attr` on your elements, so that the toolbar can leverage it for finding elements. However, since a lot of our users already used their own data attributes, we now support adding a list of your own data attributes for the toolbar to look for. 
+
+You can configure this in 'Project Settings'.
+
+### [Dashboard collaboration features](https://github.com/PostHog/posthog/pull/3756)
+
+![Dashboard collab](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/dashboard-collab.png)
+
+Our dashboards keep getting better with every new release, and this one is no different!
+
+Dashboards now support descriptions and tags, making it easier for teams to collaborate when creating internal analytics reports.
+
+These are enterprise features available to our paying Cloud customers, and to enterprise self-hosted users.
+
+If you're interested in having these features on your self-hosted PostHog instance, contact us on <i>sales@posthog.com</i>.
+
+### [Session recording for heavy websites](https://github.com/PostHog/posthog/pull/3705)
+
+If you tell Karl you found an issue with session recording, he will fix it.
+
+Such was the case with session recording for heavy websites. We were failing to process heavy snapshots, leading users of these websites unable to get many session recordings.
```

Comment:
What is "heavy"?

Reply:
Yeah I opted for not defining it but maybe I should...